lib/sgn-sdk.js

```diff
@@ -73,6 +74,14 @@ if (isBrowser()) {
         }
 
         SGN.config.set(config);
+
+        if (component === 'paged-publication-viewer') {
+            SGN.CoreUIKit.PagedPublication(scriptEl).render();
+        }
+
+        if (component === 'list-paged-publications') {
+            SGN.CoreUIKit.ListPublications(scriptEl).render();
```

Comment:
I would prefer using `CoreUIKit.ListPublications` here(and `CoreUIKit.PagedPublication` above), we try not to use the singleton within the SDK as it will make it difficult to component-ize the SDK when we get around to it.